py/trtorch/csrc/tensorrt_classes.h

```diff
@@ -94,10 +94,18 @@ struct CompileSpec : torch::CustomClassHolder {
     input_ranges.push_back(*ir);
   }
 
+  int64_t getPTQCalibratorHandle() {
+    return (int64_t)ptq_calibrator;
```

Comment:
Can you let me know why we do this cast to int64_t and type cast it back in the `setPTQCalibratorViaHandle` call ?

Reply:
TorchBind cannot handle pointers as arguments so this was the cheapest way to get a pointer added to the struct. we get the int64_t casted pointer from the original struct and there is `void setPTQCalibratorViaHandle(int64_t handle)` to set the pointer from an int64_t in a struct owned by torchbind

Reply:
Both these functions dont get exposed to the user, they are purely used internally